extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/MSQDestination.java

```diff
@@ -21,11 +21,15 @@
 
 import com.fasterxml.jackson.annotation.JsonSubTypes;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import org.apache.druid.msq.indexing.destination.DataSourceMSQDestination;
+import org.apache.druid.msq.indexing.destination.DurableStorageDestination;
+import org.apache.druid.msq.indexing.destination.TaskReportMSQDestination;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 @JsonSubTypes(value = {
     @JsonSubTypes.Type(name = DataSourceMSQDestination.TYPE, value = DataSourceMSQDestination.class),
-    @JsonSubTypes.Type(name = TaskReportMSQDestination.TYPE, value = TaskReportMSQDestination.class)
+    @JsonSubTypes.Type(name = TaskReportMSQDestination.TYPE, value = TaskReportMSQDestination.class),
+    @JsonSubTypes.Type(name = DurableStorageDestination.TYPE, value = DurableStorageDestination.class)
```

Comment:
```suggestion
    @JsonSubTypes.Type(name = DurableStorageMSQDestination.TYPE, value = DurableStorageDestination.class)
```

nit: For consistency with the other two names, this class could be renamed